tokio-io/src/codec/mod.rs

```diff
@@ -17,7 +17,9 @@ mod lines_codec;
 
 pub use self::decoder::Decoder;
 pub use self::encoder::Encoder;
+#[allow(deprecated)]
```

Comment:
why?

Reply:
When I started to deprecate Encode and Decoder (which I must have reverted before pushing), BytesCodec started to complain that it was using deprecated items.

But this can probably be put in the BytesCodec file.